docs/api/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.3
+
+info:
+  contact:
+    email: harvester@dans.knaw.nl
+  description: >-
+    Voor het eenvoudig ophalen van een publicatie of van data met een in Nederland toegekende NBN is er de <a href
+    ='https://persistent-identifier.nl'>Nationale Resolver</a>.<br>Deze wordt beheerd door <a href
+    ='https://dans.knaw.nl'>Data Archiving and Networked Services</a> [DANS] en <a href="https://www.kb.nl">KB nationale bibliotheek</a> [KB] <br> Meer informatie over de <a href="https://www.kb.nl/organisatie/onderzoek-expertise/informatie-infrastructuur-diensten-voor-bibliotheken/registration-agency-nbn">'Registration Agency NBN'</a>.
+  license:
+    name: Apache License 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  title: NBN:Reslover
+  version: 0.1.0
+
+externalDocs:
+  description: NBN Resolver op GitHub
+  url: 'https://github.com/DANS-KNAW/gmh-meresco/tree/brigmh'
+
+paths:
+
+  /nbn:
+    post:
+      summary: 'Registers a new URN:NBN identifier with associated locations.'
+      description: 'Registers a new URN:NBN associated with a prioritized list of locations. Multiple locations are prioritized in respective order. The first location is the preferred. The second is failover, etc. <br>The identifier must have a prefix that matches the authenticated user.'
+      operationId: 'createNbnLocations'
+      requestBody:
+        required: true
+        description: A json object that contains the URN:NBN and assosiated locations.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NbnLocationsObject'
+      responses:
+        '201':
+          description: Successful operation (created new)
+        '400':
+          description: Invalid URN:NBN identifier or location(s) supplied
```

Comment:
* what happens when the urn:nbn is already registered? 
* or when the urn:nbn has a prefix not from this authenticated user? maybe a 403 in that case
* what are invalid locations? non-URI's?

Reply:
1. HTTP 409 Conflict
 1. HTTP 403 Forbidden
 1. Correct, changed description.